src/main/java/org/opensearch/neuralsearch/processor/combination/ScoreCombinationUtil.java

```diff
@@ -117,7 +117,7 @@ protected void validateIfWeightsMatchScores(final float[] scores, final List<Flo
      * @param weightsList
      */
     private void validateWeights(final List<Float> weightsList) {
-        boolean isOutOfRange = weightsList.stream().anyMatch(weight -> !Range.between(0.0f, 1.0f).contains(weight));
+        boolean isOutOfRange = weightsList.stream().anyMatch(weight -> !Range.of(0.0f, 1.0f).contains(weight));
```

Comment:
Why this **between** is changed to **of** in Range?

Reply:
`between` method in deprecated in apache commons code https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/Range.html#between(T,T)